### PR TITLE
Update realtimeUnsubscribeFromVolumeIndicator to receive callback param

### PR DESCRIPTION
**Pull request by bradmanAWS**
**Issue #:** 
https://github.com/aws/amazon-chime-sdk-component-library-react/issues/280

**Description of changes:**
- Update the `realtimeUnsubscribeFromVolumeIndicator` method to optionally receive a `callback` param to match the interface updated in the `amazon-chime-sdk-js@2.4`.
- Update the meeting demo application to use the new callback param which no longer causes active speaker detection to stop after closing the roster.
**Testing**
1. Have you successfully run `npm run build:release` locally?
Yes
2. How did you test these changes?
Updated the demo application and saw active speaker detection working after unsubscribing from volume indicators (closing the roster).
3. If you made changes to the component library, have you provided corresponding documentation changes?
Yes
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -109,7 +109,7 @@
     "webpack-cli": "^3.3.2"
   },
   "peerDependencies": {
-    "amazon-chime-sdk-js": "1.x || 2.x",
```
Comment (p-foucht) [inline package.json:112]: Hm.. We don't want to force customers to use only 2.4. We really want 2.4 and up, but now that I think about it, I'm unsure if you can specify that. We may have to just do 2.x here

Comment (bradmanAWS) [reply package.json:112]: Good links! Updated.

Comment (p-foucht) [reply package.json:112]: `>=2.4.x` would upgrade to the next major version (3.x, 4.x, etc), which could break builders' applications as well

I think we want something like this? (Probably good to double check the link I sent you)

`>=2.4 <3.0`

Maybe @davejlane can confirm if this is the right approach

Comment (bradmanAWS) [reply package.json:112]: Ok, I'm surprised we would need to worry about 3.0 before it exists, but I can take a deeper look into the standards in those docs.

**Pull request merged**